src/components/Reusable/Form.js

```diff
@@ -0,0 +1,11 @@
+
+const Form = (props) => {
+
+    return (
+        <form className='flx flx-column mg-left-lg mg-right-lg'>
+            {props.children}
+        </form>
+    )
+}
```

Comment:
Here you can directly destructure the children prop.

Reply:
made the changes. thanks,